app/routes/chatbot.js

```diff
@@ -13,13 +12,24 @@ const phoenix = require('../../lib/phoenix');
 const stathat = require('../../lib/stathat');
 const ClosedCampaignError = require('../exceptions/ClosedCampaignError');
 
+// config modules
+const requiredParamsConf = require('../../config/middleware/chatbot/required-params');
+const requestContextConf = require('../../config/middleware/chatbot/request-context');
```

Comment:
What do you think about renaming this to something like `request-input-params`? `Context` feels so generic, but I guess it brings up the question of maybe it's the place to store loading a Campaign per an incoming Broadcast ID too.